### PR TITLE
Element-R: silence log errors when viewing a pending event

### PR DESCRIPTION
**Pull request by richvdh**
Fixes the second half of vector-im/element-web#26272

---
Here's what your changelog entry will look like:

## 🐛 Bug Fixes
 * Element-R: silence log errors when viewing a pending event ([\#3824](https://github.com/matrix-org/matrix-js-sdk/pull/3824)).

### CODE REVIEW

File: spec/integ/crypto/crypto.spec.ts [hunk 1]
```diff
@@ -1700,6 +1706,106 @@ describe.each(Object.entries(CRYPTO_BACKENDS))("crypto (%s)", (backend: string,
         await sendEventPromise;
     });
 
+    describe("getEncryptionInfoForEvent", () => {
+        it("handles outgoing events", async () => {
+            aliceClient.setGlobalErrorOnUnknownDevices(false);
+            //aliceClient.getCrypto()!.globalErrorOnUnknownDevices = false;
```
Comment (florianduros) [inline spec/integ/crypto/crypto.spec.ts:1712]: Remaining commented code ?

**Review by florianduros — APPROVED** [review]
LGTM

File: spec/integ/crypto/crypto.spec.ts (hunk 1 above)
Comment (richvdh) [reply spec/integ/crypto/crypto.spec.ts:1712]: oops, thanks

Comment (richvdh) [reply spec/integ/crypto/crypto.spec.ts:1712]:
```suggestion
```

**Pull request merged**